src/components/WebComponent/WebComponent.scss

```diff
@@ -2,11 +2,16 @@
 @use '../../font-size' as *;
 @use '../../colours' as *;
 
+:host {
+  font-size: 1.6rem;
+}
+
 #wc {
-  @include font-size-1(regular);
+  background: var(--editor-secondary-theme, transparent); // you can also access root variables here too
```

Comment:
Can we switch this is a sass variable somehow?

Reply:
This is the same as below and annoyingly I don't think so